doc/configuration.md

```diff
@@ -6,157 +6,144 @@ features have been added, including historic rewards with Shelley, scripts with
+```
+# <-- Rest of configuration -->
+# ...
+insert_options:
+  tx_out:
+    value: enable
+    force_tx_in: false
+  ledger: enable
+  shelley:
+    enable: true
+  multi_asset:
+    enable: true
+  metadata:
+    enable: true
+  plutus:
+    enable: true
+  governance: enable
+  offchain_pool_data: enable
+  json_type: text
+```
 
-Disables the inputs and outputs. With this flag
-- `tx_in` table is left empty
-- `tx_out` table is left empty
-- `ma_tx_out` table is left empty
-- `tx.fee` has a wrong value 0
-- `redeemer.script_hash` is left Null
+## Properties
 
-It's similar to `--bootstrap-tx-out` except the UTxO is never populated. However after using this
-flag db-sync can be stopped and restarted with `--bootstrap-tx-out` to load the UTxO from the
-ledger.
+`insert_options` may contain the following elements:
 
-### --disable-shelley : Experimental
+| Property             | Type       | Required |
+| :------------------- | :--------- | :------  |
+| preset               | `enum`     | Optional |
+| tx\_out              | `object`   | Optional |
+| ledger               | `enum`     | Optional |
+| shelley              | `object`   | Optional |
+| multiasset           | `object`   | Optional |
+| metadata             | `object`   | Optional |
+| plutus               | `object`   | Optional |
+| governance           | `enum`     | Optional |
+| offchain\_pool\_data | `enum`     | Optional |
+| json_type            | `enum`     | Optional |
 
-Disables the data related to shelley, all certificates, withdrawalsand  param proposals.
-Doesn't disable `epoch_stake` and `rewards`, For this check `--disable-ledger`.
+### Preset
 
-### --disable-multiassets : Experimental
+Preset is an aggregate setting that overrides all other properties. For example, setting
+preset to `"full"` will enable all insert options.
 
-Disables the multi assets tables and entries.
+`preset`
 
-### --disable-metadata : Experimental
+ * Type: `string`
 
-Disables the tx_metadata table.
+**enum**: The value of this property must be equal to one of the following values:
 
-### --disable-plutus-extra : Experimental
+| Value           | Explanation                                                  |
+| :-----------    | :----------------------------------------------------------- |
+| `"full"`        | Enable all options                                           |
+| `"only_utxo"`   | Only load `block`, `tx`, `tx_out` and `ma_tx_out`            |
+| `"only_gov"`    | Disable most data except governance data.                    |
+| `"disable_all"` | Only load `block`, `tx` and data related to the ledger state |
 
-Disables most tables and entries related to plutus and scripts.
+**Full**
 
-### --disable-offchain-pool-data : Experimental
-_(previousle called --disable-offline-data)__
+Enables all other options.
 
-Disables fetching pool offchain metadata.
+**Only UTxO**
+
+This is the equivalent of using `--dont-use-ledger` `--disable-shelley`,
```

Comment:
might want to mention that this refers to command line flags we previously had.

> The equates to the following deprecated flags:  `--dont-use-ledger` `--disable-shelley`...

Reply:
I believe the docs should basically be finished now.